include/mouse.h

```diff
@@ -134,7 +134,7 @@ bool MOUSEPS2_SendPacket();
 // ***************************************************************************
 
 void MOUSEDOS_BeforeNewVideoMode();
-void MOUSEDOS_AfterNewVideoMode(const bool setmode);
+void MOUSEDOS_AfterNewVideoMode(const bool is_mode_set);
```

Comment:
Very helpful rename! The original `setmode` a bool appears to _ask_ or _tell_ the function to set some mode.

Reply:
Actually, I think it is still misleading. The parameter is set to true is the screen mode is changed and to false if this is driver reset. I'll have to think about better name - just not today, it's rather late here.

Reply:
if it relates to the `bios_screen_mode` variable below, then perhaps `bios_screen_changed` would be a consistent match.

Reply:
I have changed to `is_mode_changing` - this will indicate that we are a part of the screen mode change flow.